xarray/core/variable.py

```diff
@@ -957,7 +969,11 @@ def chunk(self, chunks=None, name=None, lock=False):
 
         return type(self)(self.dims, data, self._attrs, self._encoding, fastpath=True)
 
-    def isel(self, indexers=None, drop=False, **indexers_kwargs):
+    def isel(
+        self: VariableType,
+        indexers: Mapping[Hashable, Any] = None,
+        **indexers_kwargs: Any
+    ) -> VariableType:
```

Comment:
Did we lose the `drop` keyword? Is that intentional?

Reply:
Yes, it was completely useless and untested. I'll add it to the What's New as a breaking change unless somebody can come up with reasoning as to why it was there...

Reply:
I think this was a bad copy/paste. I agree, this argument only makes sense on Dataset/DataArray.

Reply:
Added to What's New